### ISSUE TITLE
iOS privacy manifest

### ISSUE THREAD

**Issue by gkasireddy202** [ivpusic/react-native-image-crop-picker#2040]
### Version
Tell us which versions you are using: 

- react-native-image-crop-picker v0.40.0
- react-native v0.68.7

### Platform
Tell us to which platform this issue is related

- iOS

### Expected behavior
Starting May 1, Apple will require iOS submissions to the App Store to have privacy manifests describing the use of required reason APIs: https://developer.apple.com/news/?id=3d8a9yyh.
Need the iOS privacy manifest file for this library.

### Actual behaviour

No updated iOS privacy manifest file

### Steps to reproduce

1.

2.

3.

### Attachments

// stacktrace or any other useful debug info

Love react-native-image-crop-picker? Please consider supporting our collective:
👉  https://opencollective.com/react-native-image-crop-picker/donate

**Comment by gkasireddy202** [comment]
Any update on this?

**Comment by Maker-Mark** [comment]
@ivpusic Are you aware of this/onto the issue? The reasons needed to be added to the `PrivacyInfo.xcprivacy` file are listed in. If these changes are not updated all users of this package will not be able to submit after may 1st to app store 🙏 
https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api?language=objc

**Comment by CatChen** [comment]
This repo probably just need to declare this one:

> 3B52.1
> Declare this reason to access the timestamps, size, or other metadata of files or directories that the user specifically granted access to, such as using a document picker view controller.

It could be as simple as:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>NSPrivacyAccessedAPITypes</key>
	<array>
		<dict>
			<key>NSPrivacyAccessedAPIType</key>
			<string>NSPrivacyAccessedAPICategoryFileTimestamp</string>
			<key>NSPrivacyAccessedAPITypeReasons</key>
			<array>
				<string>3B52.1</string>
			</array>
		</dict>
	</array>
</dict>
</plist>
```

**Comment by ivpusic** [comment]
fix released in v0.41.0
